Dockerfile

```diff
@@ -1,16 +1,13 @@
-FROM alpine:3.13.5@sha256:def822f9851ca422481ec6fee59a9966f12b351c62ccb9aca841526ffaa9f748
+FROM alpine:3.14.1@sha256:be9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c
 
 USER root
 
 ENTRYPOINT ["tini", "--"]
 
-RUN ["apk", "--no-cache", "upgrade"]
-RUN ["apk", "--no-cache", "add", "tini", "dnsmasq", "bash", "curl", "openssl"]
-# naxsi and awscli are not available from the main Alpine repositories yet.
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing", "add", \
-     "nginx-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-naxsi=1.16.1-r0", "nginx-naxsi-mod-http-xslt-filter=1.16.1-r0"]
+RUN ["apk", "--no-cache", "-U", "upgrade"]
```

Comment:
I think -U and --no-cache don't work correctly together, -U says to update the cache first, but --no-cache says to not use the cache, I'm not sure if the side effect here is to mean -U is ignored, or if we end up storing a cache in a layer. I think it's just ignored since if I build the image with and without the -U the size is identical, but if I build it without the --no-cache then it's a few MB bigger.

Reply:
I guess we can take out the `-U` for this.